src/container/Home/index.js

```diff
@@ -38,13 +71,41 @@ class Home extends Component {
         )
         : reciters;
 
+    const options = [
+      {
+        key: 1,
+        label: 'Recitations',
+      },
+      {
+        key: 2,
+        label: 'Recitations from Haramain Taraweeh',
+      },
+      {
+        key: 3,
+        label: 'Non-Hafs Recitations',
+      },
+      {
+        key: 4,
+        label: 'Recitations with Translations',
+      }
+
+    ];
```

Comment:
Maybe we can move this to a constants file, something like `SEARCH_FILTER_OPTIONS`